src/Configuration/Dotenv/Parse.hs

```diff
@@ -85,8 +85,9 @@ interpolatedValueCommandInterpolation = do
       symbol = L.symbol sc
 
 literalValueFragment :: String -> Parser VarFragment
-literalValueFragment charsToEscape = VarLiteral <$> some (escapedChar <|> normalChar)
+literalValueFragment charsToEscape = VarLiteral <$> some (newlineChar <|> escapedChar <|> normalChar)
   where
+    newlineChar  = string "\\n" >> return '\n'
```

Comment:
@FranzGB I see `normalChar` parses any char that doesn't need escape. Do you know why that doesn't include the `\n`?

Reply:
Hi Cris! Yes, basically this line `escapedChar = (char '\\' *> anySingle) <?> "escaped character"` is eating all the `\` not treating `\n` as a single character and leaving just the n as that is a normalChar.

Reply:
I do have another idea on how could we achieve this without adding a new function here `(newlineChar <|> escapedChar <|> normalChar)` but it will have to require modifying `escapedChar` function. Maybe we can catch up on this WDYT?